packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,277 @@
+import { GEL_GROUP_CD_MIN_WIDTH } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_WHITE,
+  C_POSTBOX,
+  C_SHADOW,
+  C_EBON,
+  C_METAL,
+} from '@bbc/psammead-styles/colours';
+import { string, oneOf, shape, node } from 'prop-types';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const BulletinWrapper = styled.div`
+  background-color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  position: relative;
+  text-decoration: none;
+
+  @media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem) {
+    display: none;
+  }
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${C_WHITE};
+    fill: currentColor;
+    margin-right: ${GEL_SPACING};
+    height: ${GEL_SPACING_DBL};
+  }
+`;
+
+const PlayCTA = styled.div.attrs({ 'aria-hidden': true })`
+  background-color: ${({ isLive }) => (isLive ? C_POSTBOX : C_EBON)};
+  color: ${C_WHITE};
+  width: 100%;
+  display: block;
+  text-align: center;
+  padding: ${GEL_SPACING};
+  vertical-align: middle;
+`;
+
+const StyledSummary = styled.p`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => getSansRegular(service)}
+  color: ${C_SHADOW};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};
+`;
+
+const StyledHeading = styled.h3`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => getSansBold(service)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+`;
+
+const HeadingLink = styled.a`
+  position: static;
+  color: ${C_EBON};
+  text-decoration: none;
+
+  &:before {
+    bottom: 0;
+    content: '';
+    left: 0;
+    overflow: hidden;
+    position: absolute;
+    right: 0;
+    top: 0;
+    white-space: nowrap;
+    z-index: 1;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+
+  :hover,
+  :focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledLiveIndicator = styled.span`
+  color: ${C_POSTBOX};
+  text-transform: uppercase;
+  text-decoration: none;
+  :hover {
+    text-decoration: none;
+  }
+`;
+
+const BulletinBodyWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+`;
+
+const BulletinImageWrapper = styled.div`
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledBulletinImage = styled.img`
+  display: block;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+`;
+
+const BulletinBody = ({
+  title,
+  summary,
+  type,
+  liveText,
+  script,
+  service,
+  link,
+  CTAText,
+  hiddenTitleText,
+  dir,
+}) => (
+  <BulletinBodyWrapper dir={dir}>
+    <StyledHeading script={script} service={service}>
+      <span>
+        <VisuallyHiddenText>{hiddenTitleText}</VisuallyHiddenText>
+        {liveText && <LiveIndicator text={liveText} />}
```

Comment:
I’m not sure about tying the state of ‘isLive’ to whether or not the translation copy is being passed in, would it make more sense to always pass in copy that's coming from translation bundles and have a separate prop to toggle the live state?

Reply:
Ah yes, I wondered about this too. The reason I went for this is that the text in the `liveText` prop gets styled differently to the rest of the title and if the prop is live then the colour of the call to action also changes. If there were a separate prop `isLive` and liveText were always shown we could have a non-live CTA with `liveText` displayed or vice-versa which seems like it shouldn't be possible to me. What are your thoughts about that case?

Reply:
I agree with your reasoning! I would keep the logic exactly the same, but where you're checking for `liveText`, check instead to see if the `isLive` prop is true. Then you can just pass `liveText` in as a translation string (rather than setting it to `const liveText = isLive ? 'Live ' : null;` outside of the component). So you'd keep the condition `{isLive && <LiveIndicator text={liveText} />}`, and only render the LiveIndicator when `isLive` is true.

Reply:
Gotcha! Sounds like a sensible idea to me so have implemented it.